packages/gel/src/options.ts

```diff
@@ -46,17 +46,15 @@ export interface SimpleRetryOptions {
 export type WarningHandler = (warnings: errors.GelError[]) => void;
 
 export const throwWarnings: WarningHandler = (warnings) => {
-  throw new Error(
-    `warnings occurred while running query: ${warnings.map((warn) => warn.message)}`,
-    { cause: warnings },
+  throw new AggregateError(
+    warnings,
+    `Warnings occurred while running query: ${warnings.map((warn) => warn.message)}`,
```

Comment:
Let's keep the original casing here for better backwards compat. (see the failing tests, but also some folks could've stumbled into this already?)